packages/block-editor/src/components/provider/use-media-upload-settings.js

```diff
@@ -17,6 +17,7 @@ function useMediaUploadSettings( settings = {} ) {
 			mediaSideload: settings.mediaSideload,
 			maxUploadFileSize: settings.maxUploadFileSize,
 			allowedMimeTypes: settings.allowedMimeTypes,
+			typesNotSupportedByServer: settings.typesNotSupportedByServer,
```

Comment:
Note that this hook here is for updating the settings for the new client-side media processing functionality.

It probably doesn't need to know whether the server supports resizing the images because the resizing can happen in the browser and it can automatically convert the image to something else anyway.

If we think it's needed though, then this requires some updates in the `upload-media` package too, which is probably best left for another PR.

Reply:
Ok, will remove for now, I didn't understand what this was used for. Thanks for pointing this out.

Reply:
removed in https://github.com/WordPress/gutenberg/pull/68644/commits/7c2e8b6cb76deac2f83cdcccb9ae50c1c7695cbb